sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient_extensions.cs

```diff
@@ -0,0 +1,367 @@
+        /// <param name="group">Target group name, which length should be greater than 0 and less than 1025.</param>
+        /// <param name="message"></param>
+        /// <param name="excluded">Excluded connection Ids</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Response"/> if successful.</returns>
+        public virtual async Task<Response> SendToGroupAsync(string group, string message, IEnumerable<string> excluded = null, CancellationToken cancellationToken = default)
+        {
+            RequestOptions options = default;
+            if (cancellationToken != default)
+                options = new RequestOptions() { CancellationToken = cancellationToken };
+
+            return await SendToGroupAsync(group, JsonContent, RequestContent.Create(message), excluded, options).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Send message to a group of connections.
+        /// </summary>
+        /// <param name="group">Target group name, which length should be greater than 0 and less than 1025.</param>
+        /// <param name="message"></param>
+        /// <param name="excluded">Excluded connection Ids</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Response"/> if successful.</returns>
+        public virtual Response SendToGroup(string group, string message, IEnumerable<string> excluded = null, CancellationToken cancellationToken = default)
+        {
+            RequestOptions options = default;
+            if (cancellationToken != default)
+                options = new RequestOptions() { CancellationToken = cancellationToken };
+
+            return SendToGroup(group, JsonContent, RequestContent.Create(message), excluded, options);
+        }
+
+        /// <summary> Check if there are any client connections inside the given group. </summary>
+        /// <param name="group"> Target group name, which length should be greater than 0 and less than 1025. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual async Task<Response<bool>> GroupExistsAsync(string group, CancellationToken cancellationToken = default)
+        {
+            var options = new RequestOptions() { StatusOption = ResponseStatusOption.NoThrow, CancellationToken = cancellationToken };
+            var response = await GroupExistsAsync(group, options).ConfigureAwait(false);
+            return Response.FromValue(response.Status == 200, response);
+        }
+
+        /// <summary> Check if there are any client connections inside the given group. </summary>
+        /// <param name="group"> Target group name, which length should be greater than 0 and less than 1025. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual Response<bool> GroupExists(string group, CancellationToken cancellationToken = default)
```

Comment:
Fine for now, but longer term would we want to try to teach the code generator to just spit a method that returns `Response<bool>` here?  It feels like we could provide metadata on these operations that would tell the code generator to generate a method that returned Response<bool> instead of just Response in these cases.  I feel like what the code generator spits out today for any sort of "XyzExists" style operation will immediately need a helper like this to be palatable to customers.